Core/Commands/EnterRavencrestCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using PathOfTerraria.Common.Subworlds;
+using SubworldLibrary;
+
+namespace PathOfTerraria.Core.Commands;
+
+#if DEBUG
+public sealed class EnterRavencrestCommand : ModCommand
```

Comment:
Thought we had a command for this already? Sec looking

Reply:
Yea this one: https://github.com/Path-of-Terraria/PathOfTerraria/blob/ec9d77c79eab943baec88042fd4af0c904224201/Core/Commands/TravelCommand.cs

So `/travel Ravencrest` works

Reply:
Yeah, I had spent a bit without realizing there was the Travel command. That being said, it uses the non-Subworld WorldNavigation system, which having used it a bit I'd be more inclined to deprecate than to keep updating. Replacing the current world is cool, but it's just less convenient, harder and I have no idea how it'd behave in multiplayer. It also adds in world files to the user's Mods directory, which after playing becomes garbage, which I'd like to avoid.

Reply:
Right, it also has a chance to break the map and if using the Witch's Broom item will unconditionally crash. There'd be a lot of edge cases we have to account for that are entirely circumvented with subworlds.

Reply:
Alright, `/travel SubworldName` will now teleport you to the given subworld. You can put in `EyeDomain` or `Eye` as a parameter, `Domain` or `Subworld` are an optional part of the string. So, `/travel Ravencrest` is going to take you to the subworld now. 
As such, I've removed the SpawnWorldCommand (`/newworld`) and the new ravencrest command as they're useless, assuming we're fine with getting rid of the WorldNavigation system.